### ISSUE TITLE
Build fail when integrated into a project that has a ObjC bridging header

### ISSUE THREAD

**Issue by jilouc** [teads/TeadsSDK-iOS#121]
Since upgrading to 4.6.2 (with Xcode 11.4.x), we are facing a build error in our project:

```
<unknown>:0: error: using bridging headers with module interfaces is unsupported
```

I have been able to create a very simple project reproducing the issue at:
https://github.com/jilouc/TestTeads

It seems that as soon as you add an Objective-C bridging header to the project, the build starts failing. 

It also happens when integrated manually if we enable `BUILD_LIBRARY_FOR_DISTRIBUTION`. So it doesn't seem to be a Cocoapods issue.

**Comment by jilouc** [comment]
We worked around Teads podspec setting `BUILD_LIBRARY_FOR_DISTRIBUTION` using a Cocoapods post_install hook :

```ruby
post_install do |installer|
  installer.aggregate_targets.each do |aggregate_target|
    aggregate_target.xcconfigs.each do |config_name, config_file|
      config_file.attributes['BUILD_LIBRARY_FOR_DISTRIBUTION'] = 'NO'
      xcconfig_path = aggregate_target.xcconfig_path(config_name)
      config_file.save_as(xcconfig_path)
    end
  end
end
```

This way the build completes, but now we have trouble uploading to the AppStore. It is apparently some Bitcode issue. Here's what we received from Apple:

> ITMS-90562: Invalid Bundle - There's an issue with the app's bitcode compilation. Rebuild the app using the current public (GM) version of Xcode and upload it again.
> 
> ITMS-90562: Invalid Bundle - The app submission can not be successfully recompiled from bitcode due to missing symbols during linking. You can try to reproduce and diagnose such issues locally by following the instructions from: https://developer.apple.com/library/archive/technotes/tn2432/_index.html

Following the troubleshooting steps in Apple documentation we get this error:
```
ipatool failed with an exception: 
#<CmdSpec::NonZeroExitException: $ /Applications/Xcode.app/Contents/Developer/usr/bin/python3 
/Applications/Xcode.app/Contents/Developer/usr/bin/bitcode-build-tool -v -t 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin 
-L /var/folders/24/lv2nv0v95l748x6x02n812gw0000gn/T/ipatool20200428-95813-i91xsg/thinned-out/arm64/Payload/XXXXX.app/Frameworks/TeadsSDK.framework 
-L /var/folders/24/lv2nv0v95l748x6x02n812gw0000gn/T/ipatool20200428-95813-i91xsg/thinned-out/arm64/Payload/XXXXX.app/Frameworks 
--sdk /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk 
-o /var/folders/24/lv2nv0v95l748x6x02n812gw0000gn/T/ipatool20200428-95813-i91xsg/thinned-out/arm64/Payload/XXXXX.app/Frameworks/TeadsAdMobAdapter.framework/TeadsAdMobAdapter 
--generate-dsym /var/folders/24/lv2nv0v95l748x6x02n812gw0000gn/T/ipatool20200428-95813-i91xsg/thinned-out/arm64/Payload/XXXXX.app/Frameworks/TeadsAdMobAdapter.framework/TeadsAdMobAdapter.dSYM 
--strip-swift-symbols /var/folders/24/lv2nv0v95l748x6x02n812gw0000gn/T/ipatool20200428-95813-i91xsg/thinned-in/arm64/Payload/XXXXX.app/Frameworks/TeadsAdMobAdapter.framework/TeadsAdMobAdapter
Status: pid 4372 exit 1
```
So it seems to fail when trying to get dsyms for TeadsSDK

**Comment by vincentsaluzzo** [comment]
Hello @jilouc

Thanks for submitting this issue. We have downloaded your sample project and we haven't been able to reproduce the same error then you. 

We have tried with Xcode 11.3.1 and Xcode 11.4.1 and we have no building issue.

If we change "Build Libraries for distribution" on project Xcode "TestTeads" to put it to Yes, we see the error. But this is not desired: this flag is only for building library and not an application so you don't have to set it to yes here.

However, we have updated ou _podspec_ definition 2 days ago to fix an issue on this. Could you please try to remove all CocoaPods cache you have and trying to re-install Teads SDK without doing anything in Podfile. (https://github.com/CocoaPods/Specs/blob/master/Specs/f/2/1/TeadsSDK/4.6.2/TeadsSDK.podspec.json)

**Comment by jilouc** [comment]
Ok I basically had to delete the entire local Cocoapods repository to get it to fetch the updated podspec (no luck with `pod cache clean --all` and `rm -rf ~/Library/Caches/Cocoapods`), 

Given how aggressively Cocoapods CDN is caching podspecs now, maybe it would have been better to release a new version for this change. Also the podspec here doesn't match the spec in the Cocoapods repo.

Tanks anyway for the heads up! 👍

**Comment by elneruda** [comment]
Thanks for your investigation @jilouc, I update repo podspecs

**Comment by elneruda** [comment]
New version has been released : [4.6.3](https://github.com/teads/TeadsSDK-iOS/releases/tag/v4.6.3)
Also for [TeadsAdMobAdapter](https://github.com/teads/TeadsSDK-adapter-admob-ios/issues/3#issuecomment-624090286)
